action/src/uploadCodeScan.ts

```diff
@@ -0,0 +1,38 @@
+import { context, getOctokit } from '@actions/github';
+import { SarifReport } from 'cfn-guard';
+import { compressAndEncode } from './compressAndEncode';
+import getConfig from './getConfig';
+enum Endpoints {
+  CodeScan = 'POST /repos/{owner}/{repo}/code-scanning/sarifs'
```

Comment:
I dunno man is using enum here is weird lol

Reply:
lol I just didnt want strings hardcoded - could go with consts.